### PR TITLE
 Added a default message for the invalid commands

### LINKED ISSUE #53: Add a default case for "I don't know that command"
## New feature
Currently the discord bot doesn't respond to unknown triggers, but it should let the user know about the command being invalid.

## Directions/Approach
https://github.com/osdc/bots/blob/6adc0b2cdd6b3258dfc32376272901e3cb5c1585/discord-bot/bot.js Add an else case. May need to convert the existing `if` statements to `else if`.

## First-time Contributors

**If it is the first time that you contribute to the bot, follow these steps:**

Write a comment in this issue thread to let other possible contributors know that you are working on this bug. For eg : `Hey all, I would like to work on this issue.`, follow [README to the bot 🔥✨](https://github.com/osdc/bots/blob/master/discord-bot/README.md) and feel free to ask anything related to this issue ✌️

### PR DESCRIPTION
**Pull request by reapedjuggler**
Added a default message for the commands which aren't valid .

Fixes #53 , Bot was not showing any response for unknown triggers. Fixed it by adding a default message whenever a invalid command is triggered.

### CODE REVIEW

**Comment by daemon1024** [comment]
@reapedjuggler Thanks for the PR ☺️

- Can you make the PR more descriptively titled?

- Please mention the issue number with the appropriate keyword. Eg. ```Fixes #<issue-number> ``` [Ref](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue)

**Comment by reapedjuggler** [comment]
Changed the title of the issue and provided a brief description :)

**Comment by daemon1024** [comment]
@reapedjuggler It seems like you updated the commit messages with the references. That's great. but Can you also update the Pull Request title as well as Pull Request description with reference to appropriate issue number like you did in the commit.

File: discord-bot/bot.js [hunk 1]
```diff
@@ -48,61 +51,69 @@ bot.on('message',message => {
...
         message.channel.send(blog)
     } 
     
-    if(message.content === '!instagram')
+    else  if(message.content === '!instagram')
     {
         message.channel.send('https://tenor.com/view/dont-do-that-avengers-black-panther-we-dont-do-that-here-gif-12042935')
     }
+
+    else if(message.content.length > 0 && isValid.has(message.content) === false && message.content[0] === '!')
```
Comment (daemon1024) [inline discord-bot/bot.js]: Is the `isValid` needed? This case will only execute if there are no matches in any of the above conditions :thinking:

File: discord-bot/bot.js [hunk 2]
```diff
@@ -48,61 +51,69 @@ bot.on('message',message => {
...
 
 //Social media commands
 
 bot.on('message',message => { 
+
    if(message.content === '!website')
    {    
     const website = new Discord.MessageEmbed().setTitle('Visit our Website').setURL('https://osdc.netlify.app/')
     message.channel.send(website)
+    return false;
```
Comment (daemon1024) [inline discord-bot/bot.js]: I don't think there's a need to return anything. Can you justify this?

File: discord-bot/bot.js (hunk 1 above)
Comment (reapedjuggler) [reply discord-bot/bot.js]: Earlier i though of implementing it in a different way and hence that` isValid ` , i forgot to remove it :|

File: discord-bot/bot.js (hunk 2 above)
Comment (reapedjuggler) [reply discord-bot/bot.js]: Updated

**Comment by reapedjuggler** [comment]
Also any ref for "How to change the description of the PR" will be helpful ?

Edit : Done !

**Comment by daemon1024** [comment]
@reapedjuggler

File: discord-bot/bot.js (hunk 1 above)
Comment (reapedjuggler) [reply discord-bot/bot.js]: Updated some part of the code , let me know if there is any discrepancy left :)

File: discord-bot/bot.js (hunk 2 above)
Comment (daemon1024) [reply discord-bot/bot.js]: It's still here 👀

Comment (reapedjuggler) [reply discord-bot/bot.js]: my bad , fixing it

Comment (reapedjuggler) [reply discord-bot/bot.js]: Updated !

**Comment by daemon1024** [comment]
@reapedjuggler Did you check this out locally? Can you post a screenshot of how is the bot responding?

@Arnav-Khare Can you review it once too :)

**Review by daemon1024 — APPROVED** [review]
IMG_20201108_212316_317

Works 👍🎉

**Comment by Arnav-Khare** [comment]
LGTM...

**Review by sidntrivedi — APPROVED** [review]

**Pull request merged**